sdks/fabric/interoperation-node-sdk/src/AssetManager.ts

```diff
@@ -120,6 +122,12 @@ const createHTLC = async (
     const assetExchangeAgreementStr = createAssetExchangeAgreementSerialized(assetType, assetID, recipientECert, "");
     const lockInfoStr = createAssetLockInfoSerialized(hash, expiryTimeSecs);
 
+    //If timeoutCallback is not defined, start automated listener
+    if (typeof timeoutCallback !== "function") 
+    {
+        assetLockCallbackTimed(contract,contract.chaincodeId,assetType,assetID,recipientECert,"",hash,expiryTimeSecs);
```

Comment:
Instead of directly calling `assetLockCallbackTimed` here, you need to call `StartHTLCAssetLockListener` and pass the arguments, and one of the arguments would be the `assetLockCallbackTimed`